spring-boot-2/src/main/java/org/quickperf/web/spring/config/DatabaseConfig.java

```diff
@@ -138,4 +142,13 @@ public int getSqlExecutionThreshold(){
 	public void setSqlExecutionThreshold(int threshold){
 		this.sqlExecutionThreshold = threshold;
 	}
+
+	@ManagedAttribute
```

Comment:
I would prefer to avoid adding these things in `DatabaConfig`, because it is not related to the database, and it would become a bit confusing. Perhaps a `UrlConfig` class could be added? In this case, [this new class would have to be imported in WebLiveAutoconfiguration](https://github.com/quick-perf/quickperf-live/blob/e24b4cc5941ef850043e09234734524d9375f804/spring-boot-2/src/main/java/org/quickperf/web/spring/WebLiveAutoconfiguration.java#L28).